subsys/bluetooth/services/ias/ias_client.c

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "sys/atomic.h"
+#include <errno.h>
+#include <init.h>
+#include <stdint.h>
+#include <zephyr.h>
+#include <sys/check.h>
+#include <logging/log.h>
+#include <bluetooth/gatt.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/services/ias.h>
+
+#define BT_DBG_ENABLED IS_ENABLED(CONFIG_BT_DEBUG_IAS_CLIENT)
+#define LOG_MODULE_NAME bt_ias_client
+#include "../../common/log.h"
+
+enum {
+	IAS_DISCOVER_IN_PROGRSS,
+
+	IAS_NUM_FLAGS, /* keep as last */
+};
+
+struct bt_ias_client {
+	/* Handle for alert writes */
+	uint16_t write_handle;
```

Comment:
I would suggest to name it e.g. `alert_level_handle`

Reply:
Done